erc20/vyper/vyper-erc20-spec.ini

```diff
@@ -71,56 +79,71 @@ requires:
     andBool 0 <=Int BAL_TO    andBool BAL_TO    <Int (2 ^Int 256)
 
 [transfer-success]
-k: #execute => #halt
 statusCode: _ => EVMC_SUCCESS
 output: _ => #asByteStackInWidth(1, 32)
 log: _:List ( .List => ListItem(#abiEventLog(ACCT_ID, "Transfer", #indexed(#address(CALLER_ID)), #indexed(#address(TO_ID)), #uint256(VALUE))) )
 
 [transfer-success-1]
-storage:
-    #hashedLocation({COMPILER}, {_BALANCES}, CALLER_ID) |-> (BAL_FROM => BAL_FROM -Int VALUE)
-    #hashedLocation({COMPILER}, {_BALANCES}, TO_ID)     |-> (BAL_TO   => BAL_TO   +Int VALUE)
-    _:Map
+storage: 
+    M1:IMap => M2:IMap
+origStorage: 
+    M1
 +requires:
+    andBool select(M1, #hashedLocation({COMPILER}, {_BALANCES}, CALLER_ID)) ==Int BAL_FROM
+    andBool select(M1, #hashedLocation({COMPILER}, {_BALANCES}, TO_ID)) ==Int BAL_TO
```

Comment:
by alignment, i mean, align `==Int`, by adding some white spaces between `TO_ID))` and `==Int`.